iso8583.js

```diff
@@ -169,6 +182,8 @@ Iso8583.prototype.encode = function(message, $meta, context) {
         }
     }
     buffers.unshift(this.encodeField('mtid', message.mtid || new Buffer([])));
+    buffers.unshift(this.encodeField('header', message.header || Buffer.alloc(this.fieldFormat.header.size)));
```

Comment:
Best is to check this.fieldFormat.header.size before doing this. I know that adding zero length buffer does not affect result, but for safety is better to check this.